node-group-project/src/components/PetShow.js

```diff
@@ -25,14 +25,13 @@ const PetShow = (props) => {
   let adoptForm;
   if (applicationStatus === "pending") adoptForm = "Application status Pending"
   else {
-    adoptForm = displayForm === true ? <
-      PetAdoptionsForm pet_id={pet.id} 
+    adoptForm = displayForm === true ? 
+    < PetAdoptionsForm pet_id={pet.id} 
```

Comment:
I apologize for being so strict lol, can you remove the space between the < and the PetAdoptionsForm on line 29